larq_compute_engine/core/tests/BUILD

```diff
@@ -17,9 +16,16 @@ cc_test(
     name = "packbits_tests",
     size = "small",
     srcs = ["packbits_tests.cc"],
-    copts = ["-Iext/gtest/include"],
+    linkopts = select({
+        "@org_tensorflow//tensorflow:windows": [],
+        "//conditions:default": [
+            "-lm",
+            "-lrt",
+        ],
+    }),
```

Comment:
For some reason I had to add this in order for the (re-written) test to compile, I think because of the uniform random sampling? It's very weird. I don't understand why on earth you'd have to include linker arguments in order to successfully compile something from the C++ standard library, that seems unnecessary.